docs/sources/flow/reference/components/prometheus.exporter.windows.md

```diff
@@ -0,0 +1,223 @@
+|----------------------------|----------------|---------------------------------------------------------------|---------|----------|
+| `enabled_collectors`       | string`        | List of collectors to enable.                                |         | no       |
+
+`enabled_collectors` defines a hand-picked list of enabled-by-default
+collectors. If set, anything not provided in that list is disabled by
+default. See the [Collectors list](#collectors-list) for the default set.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`prometheus.exporter.windows` to configure collector-specific options:
+
+Hierarchy    | Name             | Description                            | Required 
+-------------|------------------|----------------------------------------|----------
+exchange     | [exchange][]     | Configures the exchange collector.     | no       
+iis          | [iis][]          | Configures the iis collector.          | no       
+text_file    | [text_file][]    | Configures the text_file collector.    | no       
+smtp         | [smtp][]         | Configures the smtp collector.         | no       
+service      | [service][]      | Configures the service collector.      | no       
+process      | [process][]      | Configures the process collector.      | no       
+network      | [network][]      | Configures the network collector.      | no       
+mssql        | [mssql][]        | Configures the mssql collector.        | no       
+msmq         | [msmq][]         | Configures the msmq collector.         | no       
+logical_disk | [logical_disk][] | Configures the logical_disk collector. | no       
+
+[exchange]: #exchange-block
+[iis]: #iis-block
+[text_file]: #textfile-block
+[smtp]: #smtp-block
+[service]: #service-block
+[process]: #process-block
+[network]: #network-block
+[mssql]: #mssql-block
+[msmq]: #msmq-block
+[logical_disk]: #logicaldisk-block
+
+### exchange block
+Name | Type     | Description | Default | Required
+---- |----------| ----------- | ------- | --------
+`enabled_list` | `string` | Comma-separated list of collectors to use, for example:`AvailabilityService,OutlookWebAccess`. Matching is case-sensetive |  | no
+
+### iis block
+Name | Type     | Description | Default | Required
+---- |----------| ----------- | ------- | --------
+`app_blacklist` | `string` | If given, an application needs to not match the blacklist regexp in order for the corresponding metrics to be reported. |  | no
+`app_whitelist` | `string` | If given, an application needs to match the whitelist regexp in order for the corresponding metrics to be reported. |  | no
+`site_blacklist` | `string` | If given, a site needs to not match the blacklist regexp in order for the corresponding metrics to be reported. |  | no
+`site_whitelist` | `string` | If given, a site needs to match the whitelist regexp in order for the corresponding metrics to be reported. |  | no
```

Comment:
Trying to make the descriptions a little shorter to help the rendering of the table:

```suggestion
Name | Type     | Description | Default | Required
---- |----------| ----------- | ------- | --------
`app_blacklist` | `string` | Regular expression of applications to ignore. |  | no
`app_whitelist` | `string` | Regular expression of applications to report on. |  | no
`site_blacklist` | `string` | Regular expression of sites to ignore. |  | no
`site_whitelist` | `string` | Regular expression of sites to report on. |  | no
```